templates/js/www

```diff
@@ -58,5 +58,5 @@ function onError(error) {
  */
 
 function onListening() {
-  debug('Listening on port ' + server.address().port);
+  debug('Listening on port ' + (server.address().port || server.address()));
```

Comment:
Thanks :) It was a typo and JS doesn't require semicolons in most places, so I didn't realize.